test/libsolidity/SemanticTest.cpp

```diff
@@ -176,7 +176,17 @@ TestCase::TestResult SemanticTest::runTest(ostream& _stream, string const& _line
 				constructed = true;
 			}
 
-			if (test.call().kind == FunctionCall::Kind::Constructor)
+			if (test.call().kind == FunctionCall::Kind::Storage)
+			{
+				test.setFailure(false);
+				bool nonEmpty = !storageEmpty(m_contractAddress);
+				bytes result(1, nonEmpty);
```

Comment:
Only used here, so maybe you can remove the variable: 
```suggestion
				bytes result(1, !storageEmpty(m_contractAddress));
```